src/Files.App/Filesystem/StorageHistory/StorageHistoryOperations.cs

```diff
@@ -126,40 +129,39 @@ public async Task<ReturnResult> Redo(IStorageHistory history)
 					if (IsHistoryNull(history))
 					{
 						foreach (var source in history.Source)
-						{
-							await operations.CreateAsync(source, progress, cancellationToken);
-						}
+							await _operations.CreateAsync(source, progress, _cancellationToken);
 					}
 					break;
 				case FileOperationType.CreateLink:
 					if (!IsHistoryNull(history))
 					{
-						await operations.CreateShortcutItemsAsync(history.Source,
-							await history.Destination.Select(item => item.Path).ToListAsync(), progress, cancellationToken);
+						await _operations.CreateShortcutItemsAsync(
+							history.Source,
+							await history.Destination.Select(item => item.Path).ToListAsync(), progress, _cancellationToken);
```

Comment:
Shouldn't you move `progress` and `_cancellationToken` to different lines?

Reply:
Fixed